test/e2e/flowaggregator_test.go

```diff
@@ -213,6 +216,73 @@ func testFlowAggregatorAntctl(t *testing.T, data *TestData) {
 			}
 		})
 	}
+
+	t.Run("antctl get flowrecords -o json", func(t *testing.T) {
```

Comment:
Change the test name to align with other tests, like `GetFlowRecordsJson`

Reply:
Done